docs/learn/getting-started.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_label: '👉 Getting Started'
+description: Introduction to the LUKSO Ecosystem - why choose LUKSO? Who is LUKSO intended for?
+---
+
+import CallToActionButton from '@site/src/components/CallToActionButton';
+
+# Welcome to LUKSO
+
+The purpose of this section is to teach you the bare minimum, so that you can start coding as soon as possible to try out everything LUKSO has to offer.
+
+For more detailed information on any topic please use search or check out our [LEARN section](../learn/introduction.md).
+
+## What is LUKSO?
+
+LUKSO is layer 1, Ethereum twin blockchain. It uses a proof-of-stake consensus algorithm, making it fundamentally the same as Ethereum at a network level but with some innovative differences, particularly in how developers build applications on the network and the building blocks you use.
+
+Here are the top 3 reasons why LUKSO is different to any other blockchain:
+
+<table><thead>
+  <tr>
+    <th>**Universal Profiles**</th>
+    <th>**NFT 2.0**</th>
+    <th>**Cultural Currencies**</th>
+  </tr></thead>
+<tbody>
+  <tr>
+    <td>Smart contract based user or creator profiles are the essential foundation for the New Creative Economy.</td>
+    <td>Flexible NFTs that are extensible and directly composable with Universal Profiles.</td>
+    <td>Extensible Token that move us beyond DeFi. The future of blockchain is social.</td>
+  </tr>
+  <tr>
+    <td>
+      <ul>
+      <li>Visual</li>
+      <li>Upgradeable</li>
+      <li>Security</li>
+      <li>Gas-less</li>
+      <li>Smart</li>
+      <li>Extensible</li>
+      </ul>
+    </td>
+    <td>
+      <ul>
+        <li>Authentic</li>
+        <li>Safer</li>
+        <li>Unlimited Metadata</li>
+        <li>Updatable</li>
+        <li>Better IDs</li>
+        <li>Iteratable</li>
+        <li>Batch Transfers</li>
+      </ul>
+    </td>
+    <td>
+      <ul>
+        <li>Safer</li>
+        <li>Token Icons</li>
+        <li>Updatable</li>
+        <li>Batch Transfers</li>
+        <li>Extensible</li>
+        <li>Notifying</li>
+        <li>Unified</li>
+      </ul>
+    </td>
+  </tr>
+</tbody>
+</table>
+
+Learn more about the unique LUKSO Standard Proposals (LSPs) in the [LUKSO STANDARDS section](../standards/introduction.md).
+
+## Where should I start?
+
+Download the 🆙 browser extension using the link below:
+
+<CallToActionButton
+    color="white"
+    target="_blank"
+    link="https://chromewebstore.google.com/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn"
+    text="Install the Universal Profile Browser Extension 🧩"
+  />
+
+Then create your first Universal Profile below:
+
+<CallToActionButton
+color="white"
+target="blank"
+link="https://my.universalprofile.cloud"
+text="Create your Universal Profile 🆙"
+/>
+
+With these two elements in place, you are ready to explore the [LUKSO ecosystem](https://www.lukso.network/ecosystem).
+
+## Ready to build?
+
+Choose your path below:
+
+<table><thead>
```

Comment:
I could have here, but my plan was to style the tables later and I think that's still possible with markdown, I wouldn't be able to style the other table the same way with classes. Because the other table did need html.

Reply:
Ok then maybe we can try to Markdown links for catching broken links, but use HTML table